yarn.lock

```diff
@@ -396,6 +396,7 @@ __metadata:
     "@wordpress/base-styles": ^4.5.0
     classnames: ^2.3.1
     enzyme: ^3.11.0
+    framer-motion: 6.2.8
```

Comment:
@arthur791004 Could you help me here? Do we need to commit yarn.lock? I think I messed up the file while re-installing the \`framer-motion\`